cdc/capture/capture.go

```diff
@@ -458,12 +465,24 @@ func (c *captureImpl) campaignOwner(ctx cdcContext.Context) error {
 			orchestrator.NewGlobalState(c.EtcdClient.ClusterID),
 			ownerFlushInterval, util.RoleOwner.String())
 		c.setOwner(nil)
-		log.Info("run owner exited", zap.Error(err))
+		if err != nil {
+			log.Warn("run owner exited with error",
+				zap.String("captureID", c.info.ID), zap.Int64("ownerRev", ownerRev),
+				zap.Error(err))
+		} else {
+			log.Info("run owner exited normally",
+				zap.String("captureID", c.info.ID), zap.Int64("ownerRev", ownerRev))
+		}
 		// if owner exits, resign the owner key
 		if resignErr := c.resign(ctx); resignErr != nil {
```

Comment:
Do we need to use a new context in case the `ctx` has cancelled?

Reply:
The capture is closed asynchronously, https://github.com/pingcap/tiflow/blob/3405b79aa2fa01076541322a8c92aea288d8c386/cdc/capture/capture.go#L553-L557

It has a chance that the `ctx` be canceled, looks have no negative effect on the `resign` call. 

How about using` context.Background()` to prevent from being canceled?